### DISCUSSION TITLE
Can't Deloy (Noob)

### DISCUSSION THREAD

**Discussion by dawghouse73** [nmelhado/league-page#216] [Q&A] [answered]
OK - I thought I was fairly handy with site deployments, but I believe it's in the old fashioned way. That said, I have gone so far as to delete my repository multiple times only to end up at the same errors. 

Warning: Detected "engines": { "node": ">=v16.15.0" } in your `package.json` that will automatically upgrade when a new major Node.js Version is released. Learn More: http://vercel.link/node-version

`sh: vite: command not found
Error: Command "npm run build" exited with 127
BUILD_UTILS_SPAWN_127: Command "npm run build" exited with 127
(NOTE: I've also had the "...existed with 1" shown before as well.

![image](https://github.com/nmelhado/league-page/assets/133917715/313b138b-1012-4da9-9426-1ae44e620e64)

I've done virtually nothing with the site. 
I've gone so far as to fork the repository, update the leagueinfo.json file only with my League ID and League Name - not updating anything else, and it still fails. 

I'm sure it's something simple, but being new to the github/vercel combination of tools, I'm not sure where else to look. 

Appreciate any help. 
Thanks.
House

**Comment by nmelhado** [comment]
Could you send a screenshot of your Vercel settings? Nothing in your commit looks off, not sure why it isn't working for you.

**Reply by dawghouse73** [reply]
Not seeing my images in my previous reply, so here they are again just in case:

![image](https://github.com/nmelhado/league-page/assets/133917715/bade3f63-bbb7-4c23-b82e-01d37976bff1)
![image](https://github.com/nmelhado/league-page/assets/133917715/75c80f6b-b7f3-4a68-b932-139e26ea5820)
![image](https://github.com/nmelhado/league-page/assets/133917715/29c965f1-0895-4170-a818-5b650793f755)
![image](https://github.com/nmelhado/league-page/assets/133917715/ff048852-47aa-438e-8fb9-dede75ad4b65)

**Reply by dawghouse73** [reply, marked as answer]
I managed to get it deployed - I believe it may have been an missing } on the League Info js page that I may have been somehow deleting or something when I was updating it with just my league ID and league name (weird). 

Managed to get it up and deployed - onto other issues (Managers page and Drafts page).

**Comment by dawghouse73** [comment]
Thanks for responding…

Everything is default at the moment. 
As I started over when I received the error
